.github/ISSUE_TEMPLATE/3.support.yml

```diff
@@ -8,12 +8,12 @@ body:
       value: |
         Please provide a searchable summary of the issue in the title above ⬆️.
 
-        ⚠️ **Don't use this form if the problem is [a bug](https://github.com/mui/mui-design-kits/issues/new?assignees=&labels=status%3A+needs+triage&template=1.bug.yml) or a [feature request](https://github.com/mui/mui-design-kits/issues/new?assignees=&labels=status%3A+needs+triage&template=2.feature.yml), use the dedicated form instead.**
+        ⚠️ **Don't use this form if the problem is [a bug](https://github.com/mui/mui-design-kits/issues/new?assignees=&labels=status%3A+waiting+for+maintainer&template=1.bug.yml) or a [feature request](https://github.com/mui/mui-design-kits/issues/new?assignees=&labels=status%3A+waiting+for+maintainer&template=2.feature.yml), use the dedicated form instead.**
```

Comment:
I did change this as well, since it would have created a new issue with the "old" label: `status: needs triage` we don't use anymore